superbuild/ConfigureExternalProjects.cmake

```diff
@@ -2,7 +2,7 @@
 #
 # medInria
 #
-# Copyright (c) INRIA 2013. All rights reserved.
+# Copyright (c) INRIA 2013 - 2019. All rights reserved.
```

Comment:
that would be really nice if it could be automated, isn't it? Not necessarily for this PR, but it could avoid to change it each year in each file :)

Reply:
Yeah of course. If you have a good idea to do this, I will happy to apply it.

Reply:
You could also leave 2013. Putting additional dates is useful when a work has been modified and you want to set a new copyright duration for the modified work, but the duration is 120 years, so the question is: do we care if medInria 2 (an **open source** project...) is protected until 2133 or 2139 ?
(not to mention that copyright notices are not even required in source code to actually claim a copyright)

Reply:
I think the good action it's to up this question to Inria/InriaSoft lawyer...